tests/unit/CountItems/CountWorker.js

```diff
@@ -213,15 +205,89 @@ describe('CountItems::CountWorker', () => {
             _creationDate: Date.now().toString(),
             website: { indexDocument: 'index.html' },
         };
+        w.getIsTransient = jest.fn((bucketInfo, cb) => cb(null, true));
+        mongoMock.setup.mockImplementationOnce(cb => cb());
+        mongoMock.close.mockImplementationOnce(cb => cb());
+        mongoMock.client.isConnected.mockImplementationOnce(() => false);
+        mongoMock.getObjectMDStats.mockImplementationOnce((_a, _b, _c, _d, cb) => cb(null, { value: 42 }));
+        w.countItems(bucketInfo, (err, results) => {
+            expect(err).toBeNull();
+            expect(results).toEqual({ value: 42 });
+            done();
+        });
+    });
+
+    test('should use real getIsTransient and not crash', done => {
+        const testSendFn = jest.fn();
```

Comment:
test name feels weird : there is no "feature" of the code to use a "real getIsTransient", it is really just that `getIsTransient` is mocked in all tests...
--> should find a name that better matches what "feature" of the code this test exercises (or take this as a hint that indeed we are mixing approaches here, and we shoudl either not use a mock in the tests -focus on e2e function, covering all cases in data-, or keep the mock to test the "counting" part and create other tests to verify `getIsTransient` works as expected)